pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/impl/RoleService.java

```diff
@@ -108,8 +108,24 @@ public void addObjectToSet(Set<Role> roles, Role t) {
         roles.add(t);
     }
 
+    public Role getRoleByName(String fenceOpenAccessRoleName) {
+        return this.roleRepository.findByName(fenceOpenAccessRoleName);
+    }
+
+    public Role save(Role r) {
+        return this.roleRepository.save(r);
+    }
+
     public Set<Role> getRolesByIds(Set<UUID> roleUuids) {
         return this.roleRepository.findByUuidIn(roleUuids);
     }
+
+    public void persistAll(List<Role> newRoles) {
```

Comment:
Sholud this return the roles? Will they have any data changed from saving (ex: auto generated ids)